changelog.d/8820.feature

```diff
@@ -0,0 +1,2 @@
+Add a config option, `push.group_by_unread_count`, which controls whether unread message counts in push notifications are defined as "total unread rooms" or "total unread messages".
```

Comment:
Maybe this is clearer?

```suggestion
Add a config option, `push.group_by_unread_count`, which controls whether unread message counts in push notifications are defined as "number of rooms with unread messages" or "total unread messages".
```

Reply:
I made a slight modification, but otherwise yes that's much clearer, thanks!